tests/test_worker_palavras_raw.py

```diff
@@ -28,6 +29,7 @@
 
 class TestPalavrasRawWorker(TaskTest):
 
+    @skipIf(not palavras_raw.palavras_installed(), 'palavras software is not installed')
     def test_should_run_only_if_language_is_portuguese(self):
```

Comment:
This test is not affected by the absence of palavras, since the PalavrasRaw worker tests for the document's language before it tries to run palavras. Maybe we should skip it anyways, but running it (even without palavras running) keeps testing if the worker is looking for the language in the right place.